esmero-php-fpm/Dockerfile.dev

```diff
@@ -0,0 +1,143 @@
+# build with:
+# sudo docker build -f Dockerfile.dev --build-arg LOCAL_IP=<your machine IP> -t esmero-php:development . --no-cache`
+# from https://www.drupal.org/docs/8/system-requirements/drupal-8-php-requirements
+FROM php:7.3-fpm-alpine
+
+# should be passed from the build command
+ARG LOCAL_IP
+
+# install the PHP extensions we need
+# postgresql-dev is needed for https://bugs.alpinelinux.org/issues/3642
+RUN set -eux; \
+    \
+	apk add --no-cache --virtual .build-deps \
+		coreutils \
+    		zlib-dev \
+		libxml2-dev \
+		libxslt-dev \
+		freetype-dev \
+		libjpeg-turbo-dev \
+		libpng-dev \
+		libzip-dev \
+		postgresql-dev \
+		autoconf \
+        g++ \
+        make \
+	; \
+	\
+	docker-php-ext-configure gd \
+		--with-freetype-dir=/usr/include \
+		--with-jpeg-dir=/usr/include \
+		--with-png-dir=/usr/include \
+	; \
+	\
+	docker-php-ext-install -j "$(nproc)" \
+		gd \
+		zip \
+		pdo_mysql \
+		mysqli \
+		xsl \
+		opcache \
+		exif \
+		pgsql \
+		pdo_pgsql \
+		bcmath \
+	; \
+	\
+	# install x-debug
+	pecl install xdebug-2.8.0; \
+	runDeps="$( \
+		scanelf --needed --nobanner --format '%n#p' --recursive /usr/local \
+			| tr ',' '\n' \
+			| sort -u \
+			| awk 'system("[ -e /usr/local/lib/" $1 " ]") == 0 { next } { print "so:" $1 }' \
+	)"; \
+	apk add --virtual .drupal-phpexts-rundeps $runDeps; \
+	apk del .build-deps
+	
+# set recommended PHP.ini settings
+
+RUN { \
+		echo 'zend_extension=/usr/local/lib/php/extensions/no-debug-non-zts-20180731/xdebug.so'; \
+		echo 'xdebug.remote_enable=1'; \
+		echo 'xdebug.remote_handler=dbgp'; \
+		echo 'xdebug.remote_port=9999'; \
+		echo 'xdebug.remote_autostart=1'; \
+		echo 'xdebug.remote_connect_back=0'; \
+		echo 'xdebug.idekey=archipelago'; \
+		echo "xdebug.remote_host=$LOCAL_IP"; \
```

Comment:
$LOCAL_IP, pretty sure can removed totally. In recent Docker Machines (i have 19.03.x but works since 18.03) this:

xdebug.remote_host = "host.docker.internal" will work better and requires no passing of arguments while building.
I mean, why would be want people to build the container just to get this running?

You can test host.docker.internal if you have doubts.
Also, should we use idekey PHPSTORM? As a hint? I'm ok with archipelago really